experimental/globaldata/src/data/core_helloworld_v1/mod.rs

```diff
@@ -0,0 +1,34 @@
+include!("bn.data.rs");
+include!("en.data.rs");
+include!("ja.data.rs");
+include!("ru.data.rs");
+
+#[macro_export]
+macro_rules! impl_core_helloworld_v1 {
+    ($provider:path) => {
+        impl DataProvider<icu_provider::hello_world::HelloWorldV1Marker> for $provider {
+            fn load(&self, req: DataRequest) -> Result<DataResponse<icu_provider::hello_world::HelloWorldV1Marker>, DataError> {
+                type DataStruct =
+                    <icu_provider::hello_world::HelloWorldV1Marker as icu_provider::DataMarker>::Yokeable;
+                static KEYS: &[&str] = &["bn", "en", "en-US", "ja", "ru"];
+                static BN: DataStruct = $crate::baked::core_helloworld_v1::data_core_helloworld_v1_bn!();
```

Comment:
This works, but it will be hard to generate because you're using the absolute path of where this will be included, e.g. it has to be included at `$crate::baked`. If we can't find a way to use relative addressing for the data file (either using `mod` or `include`), we should inline the data to step around this problem.

Reply:
The path is relative to the crate. We may need to tell databake-datagen what that path is, but this is totally an internal thing.

Reply:
It would be useful if the logic we add here is also usable by datagen clients, not only for this crate.